lineman/app/services/modal_service.coffee

```diff
@@ -0,0 +1,25 @@
+angular.module('loomioApp').factory 'ModalService', ($modal, $rootScope) ->
+  new class ModalService
```

Comment:
I think this looks pretty good!  May I suggest that it seems like a case for inheritance so it can follow open/closed principal (Closed for modification, open for extension). 

InvitePeopleModal and StartThreadModal inherit from BaseModal, and their unique configs live in the sub classes.